lib/api/core/entrypoints/embedded/index.js

```diff
@@ -281,7 +285,7 @@ class EmbeddedEntryPoint extends EntryPoint {
     let
```

Comment:
nitpicking : usage of string interpolation rather than concatenation
(not here but line 356 and 315)

Reply:
fixed